humble/distribution.yaml

```diff
@@ -1199,6 +1199,27 @@ repositories:
       url: https://github.com/ros-sports/gc_spl.git
       version: humble
     status: developed
+  generate_parameter_library:
+    doc:
+      type: git
+      url: https://github.com/PickNikRobotics/generate_parameter_library-release.git
+      version: main
+    release:
+      packages:
+      - generate_parameter_library
+      - generate_parameter_library_example
+      - generate_parameter_library_py
+      - parameter_traits
+      - tcb_span
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/PickNikRobotics/generate_parameter_library-release.git
+      version: 0.2.2-1
+    source:
+      type: git
+      url: https://github.com/PickNikRobotics/generate_parameter_library-release.git
```

Comment:
I think this line is supposed to be `https://github.com/PickNikRobotics/generate_parameter_library.git`

Reply:
fixed, thank you for catching that